runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkExecutionEnvironments.java

```diff
@@ -171,4 +176,12 @@ public static StreamExecutionEnvironment createStreamExecutionEnvironment(
 
     return flinkStreamEnv;
   }
+
+  private static void applyLatencyTrackingInterval(
+      ExecutionConfig config, FlinkPipelineOptions options) {
+    long latencyTrackingInterval = options.getLatencyTrackingInterval();
+    if (latencyTrackingInterval >= 0) {
```

Comment:
Will this work? The default is `-1`, so this condition will be `false`, i.e. we never disable latency tracking in Flink which is the original point of this PR.

I think we can just remove the check. What do you think?

Reply:
Idea was to stick to default flink configuration, unless user intend to overwrite it. With this PR I have a choice to disable it by passing to runner `--latencyTrackingInterval=0`. 
Did not wanted to disable it for all by default. Should I?

Reply:
I think we should disable by default. This is also what will happen for the next Flink version, I think. And it might save people some headaches if they don't have to debug it and find out the hard way, as you did.

Reply:
The default is now 0, so it gets disabled by default now. I agree with Aljoscha that the check could be removed entirely. The problem is, if you pass in negative numbers, they will simply be ignored and latency tracking will still be enabled...